benchmarks/naslib/problem.py

```diff
@@ -0,0 +1,155 @@
+import ast
+import os
+import sys
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Optional
+
+from kurobako import problem
+
+
+sys.stdout = open(os.devnull, "w")  # Suppress output
+
+
+import naslib.search_spaces as ss  # NOQA
+from naslib.search_spaces.core.graph import Graph  # NOQA
+from naslib.search_spaces.core.query_metrics import Metric  # NOQA
+import naslib.search_spaces.nasbench201.graph as nasbench201_graph  # NOQA
+import naslib.search_spaces.transbench101.graph as transbench101_graph  # NOQA
+from naslib.utils import get_dataset_api  # NOQA
+
+
+class NASLibProblemFactory(problem.ProblemFactory):
+    def __init__(self, search_space: Graph, **config: Dict[str, Any]) -> None:
```

Comment:
If possible, how about enumerating the each key and value of this `config` dict as arguments? It is easy to understand this factory. This can be applied to other classes.

Reply:
I thought the argument `config` needs to be flexible, because different benchmark class (`search_space`s) in `naslib` may have different arguments for `config`.